src/main/java/frc/robot/commands/shooter/ShootSpeaker.java

```diff
@@ -32,16 +36,21 @@ public boolean isFinished() {
 
     @Override
     public void execute() {
-        double gyro = ((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360); //Gets the gyro value 0-360
+
```

Comment:
This command's `isFinished` should be changed - your command will not properly run right now because it will end before the timer elapses.